### PR TITLE
Vec 95

### PR DESCRIPTION
**Pull request by davi17g**

### CODE REVIEW

File: basic-search/requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-aerospike-proximus==0.3.1
\ No newline at end of file
+aerospike-proximus==0.3.2.dev1
```
Comment (hev) [inline basic-search/requirements.txt:1]: This should use aerospike-vector-search client 0.5.1 now - which may break some other functionality and require input from @jdogmcsteezy.

**Review by hev — CHANGES_REQUESTED** [review]
@davi17g we have a newer client version than this one. This is probably a bigger change to make as there are some name changes involved in the API. We should track this under VEC-134.

File: prism-image-search/prism/config.py [hunk 2]
```diff
@@ -16,3 +16,4 @@ class Config(object):
     PROXIMUS_VERIFY_TLS = os.environ.get("VERIFY_TLS") or True
     PROXIMUS_MAX_RESULTS = int(os.environ.get("PROXIMUS_MAX_RESULTS") or 20)
     MAX_CONTENT_LENGTH = int(os.environ.get("MAX_CONTENT_LENGTH") or 10485760)
+    PROXIMUS_IS_LOADBALANCER = os.environ.get("PROXIMUS_IS_LOADBALANCER") or False
```
Comment (arrowplum) [inline prism-image-search/prism/config.py:19]: I think we are missing these values in the readme

**Review by arrowplum — COMMENTED** [review]
In general we should make sure the READMEs have the new configuration

File: prism-image-search/prism/config.py (hunk 2 above)
Comment (hev) [reply prism-image-search/prism/config.py:19]: I came across a few things in the README I think we should put into another PR.

File: basic-search/requirements.txt (hunk 1 above)
Comment (hev) [reply basic-search/requirements.txt:1]: Agreed. Resolving.

**Review by hev — APPROVED** [review]
Tested using docker-compose and a sandbox. All works.

**Review by arrowplum — APPROVED** [review]
Tested built against the sandbox and the published https://hub.docker.com/repository/docker/aerospike/quote-search-example/

Also experimental helm chart works fine with a few small tweaks. :+1: 

@davi17g I think you can go ahead and merge it.

**Pull request merged**